config/components/rbac/mpijob_editor_role.yaml

```diff
@@ -0,0 +1,27 @@
+# permissions for end users to edit jobs.
```

Comment:
I'm on the fence about whether to include this file in the repo. I feel like users of kubeflow should know how to set this up.
Although the same argument can be made about jobs

Reply:
I prefer to keep it, the reasons I see:
- it is consistent with what we do for Job
- it makes the integration working OOTB
- it requires knowing the Kueue labels (such as `rbac.kueue.x-k8s.io/batch-admin`)

Reply:
BatchJob is an in-tree object, so I think it's ok to maintain the yaml here. And we can better organize it like we do in `test/e2e/config`. But I don't want to block this significant feature.